test/tesselation.jl

```diff
@@ -45,5 +45,12 @@
     pts = randpoint3(10)
     pset = PointSet(pts)
     @test_throws AssertionError tesselate(pset, VoronoiTesselation(StableRNG(2024)))
+
+    # Test polygon order is the same as input points order
+    pts = randpoint2(10)
+    mesh = tesselate(pts, VoronoiTesselation(StableRNG(2024))
+    @test all(zip(pts, mesh)) do (pt, poly)
+      pt in poly || pt ∉ mesh # The pt ∉ mesh is because some points are on the border of the respective polygon and return false from `pt in poly` due to floating point precision. So if that happens we check that the point is not in any other polygon
```

Comment:
I think we can remove the randomness of this test to avoid this floating point side effect. Can you please construct a deterministic set of points where the `point in poly` predicate is true?

Reply:
Alternatively you can try to add a test that checks that the distance from the point to the `centroid(poly)` is smaller or equal to any other centroid.

Reply:
Went with the second appraoch